x-pack/test/functional/page_objects/security_page.ts

```diff
@@ -273,11 +274,7 @@ export function SecurityPageProvider({ getService, getPageObjects }: FtrProvider
 
     async addIndexToRole(index: string) {
       log.debug(`Adding index ${index} to role`);
-      const indexInput = await retry.try(() =>
-        find.byCssSelector('[data-test-subj="indicesInput0"] input')
-      );
-      await indexInput.type(index);
-      await indexInput.type('\n');
+      await comboBox.setCustom('indicesInput0', index);
```

Comment:
Attempts to address test flakyness by using the `comboBox` service to set input values, instead of whatever homegrown solution we had here. Failure screenshots indicated that the dropdown list was not dismissing properly in all cases, which caused clicks to other elements on the page to not be registered properly.